_posts/2019-10-20-iPhone-11.md

```diff
@@ -0,0 +1,45 @@
+# iPhone 11
+
+I've now spent one month using iPhone 11 as my daily driver.
+
+It is a big step up from my previous 250 EUR budget Android phone and my 5 year old iPhone 5S. This article should be seen in that light. I hadn't try a modern iOS device as my daily driver before so a lot of things that might be obvious to people using an iPhone X are not to me and I'm sure some things I point out were there in previous iPhones as well.
+
+## Pros
+
+- Great Camera (even in low light conditions)
+With both my Android phone and the 5S I had to use my wife's phone or my reflex to be able to get a non grainy picture on any light condition that was less than perfect. With iPhone 11 I don't need to worry, even at night your pictures will be more than acceptable.
+
+- "Studio Photos" are neat
+You're probably not going to use them a lot, but they're greatly done and it is a nice past time after family dinners.
+
+- Great Display
+Great natural colors and brightness. If you're used to a super saturated Samsung display you might not like the natural colors too much, but I blame Samsung here who's pushing the saturation way too high and not showing the real colors.
+
+- Good Battery Life
+More than one full day with medium use. I don't worry at all if I forget to charge it one night since my use is not heavy.
+
+- Way better apps than on Android (with a few exceptions, see "Cons")
+The software quality divide is massive. On top of the fact that a lot of great iOS apps are simply not available on Android.
+
+- Hardware Specs
+Needless to say the sheer difference in speed and responsiveness between the iPhone 11 and my previous phones is immense. There were times when my Android phone would freeze for seconds to render some heavy application. Mind that here I'm not talking about heavy games, but normal applications such as Google Music or Waze.
+
+- iOS
+The animations and scrolling on iOS is unmatched on Android. Some is due to the device itself as for the previous point, but a lot of it is just the OS. I have no idea how Google still can't do scrolling content right after all of this years, but the difference is astonishing.
+
+## Cons
+
+- Face ID is terrible
+It only works on perfect light conditions and even then only 70% of the times. Considering that it is a version 2 since version 1 was last year with the iPhone X it is totally unacceptable. Pretty much at the same level that Touch ID was with the iPhone 5S. Biggest disappointment.
```

Comment:
on - > in

time is not countable, lose the s

put since version 1 (...) in (

Reply:
Windows Hello works better

Reply:
on - > in
Consistent in my wrong ways I see..